development/print_db_data.py

```diff
@@ -122,50 +122,36 @@ def print_production_lines():
 def print_verbose_lines():
 	print('Data has been moved, this view is unavailable for now')
 	return
-	def _output_helper_prodlines(string, list):
```

Comment:
Hello @ChrisOelmueller , first point, I had to open a ticket about this. That is right. My fault. second point, but I do not really dare to write, if I write a request, please to delete this file, the file is increased since 2013 only by the year number, alone the file name print_db_data.py, who please print a database, today in 2017?
https://github.com/unknown-horizons/unknown-horizons/commits/master/development/print_db_data.py